src/modules/fancyzones/lib/ZoneWindowDrawing.cpp

```diff
@@ -9,32 +9,37 @@
 
 #include <common/logger/logger.h>
 
+namespace
+{
+    const int FadeInDurationMillis = 200;
+    const int FlashZonesDurationMillis = 700;
+}
+
 namespace NonLocalizable
 {
     const wchar_t SegoeUiFont[] = L"Segoe ui";
 }
 
 float ZoneWindowDrawing::GetAnimationAlpha()
 {
-    // Lock is being held
+    _TRACER_;
```

Comment:
I would prefer not to trace this call since it spams the log file.

Reply:
Agreed